build/Helix/runtests.cmd

```diff
@@ -28,7 +28,7 @@ echo %TIME%
 powershell -ExecutionPolicy Bypass .\InstallTestAppDependencies.ps1
 echo %TIME%
 
-set testBinaryCandidates=TerminalApp.LocalTests.dll Conhost.UIA.Tests.dll
+set testBinaryCandidates=TerminalApp.LocalTests.dll SettingsModel.LocalTests.dll Conhost.UIA.Tests.dll
```

Comment:
I uh.... sneaking this in just to clean it up?

Reply:
Yea, originally the remoting tests were local tests, and I noticed the SettingsModel local tests were missing in a bunch of places.